lib/sync/fetcher.go

```diff
@@ -97,6 +97,7 @@ func (f *BlockFetcher) fetch(ctx context.Context, si *SyncInfo) error {
 		height    = si.Height
 		nodeAddrs = si.NodeAddrs()
 	)
+	si.Bts = si.Bts[:0]
```

Comment:
This is just initialize `si.Bts`, right?

Reply:
@Charleslee522 It's that a slice shares the same backing array and capacity as the original, so the storage is reused.

Reply:
@anarcher I got it. 👍 thanks
https://tour.golang.org/moretypes/11

Reply:
@Charleslee522 Yes it is